echo_handlers.go

```diff
@@ -284,6 +284,9 @@ func (svc *Service) AppsNewHandler(c echo.Context) error {
 		sess.Save(c.Request(), c.Response())
 		return c.Redirect(302, fmt.Sprintf("/%s/auth?c=%s", strings.ToLower(svc.cfg.LNBackendType), appName))
 	}
+	if user.HubUrl != "" {
+		return c.Redirect(302, fmt.Sprintf("%s/#/apps/new%s", user.HubUrl, c.QueryString()))
```

Comment:
```suggestion
		return c.Redirect(302, fmt.Sprintf("%s/#/apps/new?%s", user.HubUrl, c.QueryString()))
```

Reply:
hmm. actually I was wondering here. thanks for double checking